pages/api/moralis/pinFile.ts

```diff
@@ -4,7 +4,7 @@ import { pinFile } from "../../../utils/moralis";
 export const config = {
   api: {
     bodyParser: {
-      sizeLimit: "5mb",
+      sizeLimit: "100mb",
```

Comment:
This will have no effect, next.js imposes a limit of 4mb

Reply:
I know bro, I tried haha